README.md

```
@@ -16,7 +16,7 @@ Give Ivan a sentence. Ivan breaks it into PR-sized tasks, **debates its own desi
 npm i -g @ariso-ai/ivan && ivan "Add rate limiting to the public API"
 ```
 
-[Quick Start](#-quick-start) · [Expert Mode](#-expert-mode-an-ai-that-argues-with-itself-so-you-dont-have-to) · [Institutional Memory](#-institutional-memory-ivan-learns-your-team) · [Commands](#-cli-reference) · [Contributing](#-contributing)
+[Quick Start](#-quick-start) · [Expert Mode](#-expert-mode-an-ai-that-argues-with-itself-so-you-dont-have-to) · [Loop Mode](#-loop-mode-keep-going-until-its-actually-good) · [Institutional Memory](#-institutional-memory-ivan-learns-your-team) · [Commands](#-cli-reference) · [Contributing](#-contributing)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)